docs/advanced_examples/LogisticRegressionTraining.ipynb

```diff
@@ -462,6 +395,339 @@
xbnOfCzkmZEi8qfERIgdC0QTBUSh4kpEiqfjx1PYunU58CZ/vZH+uw64XLVZvXoBl1/e2cPpxCk//rgQw/DHtvvn0+NWDh2azW+/baR27aYezSbeqdAjWT179mT+/Pl8/fXXBAYGMnHiRDZv3sz8+fO5+uqrL0ZGERERkQLJyvpzhuDQfHoYQAhZWSfzaRdvlJ2dgWH4kzNylZcQADIz9byQolHgkaydO3dSu3ZtDMOgffv2LF68+GLmEhERESm04OBKhIZGcPToQnLW8zrTHtzun4mMHOnhZOKk2rUvx7KOAiuBNnn0WIiPjz8REfU9G0y8VoFHsurVq8fvv/+e+3W/fv04cODARQklIiIicj5M0+Taa0diGG8D353Rmolh3IW/fxBXXjnAiXjikMsvv5oqVepimmOA1DNaN2Ga04mNHUhQUKgD6cQbFbjIOvN6poULF3L8+PEiDyQiIiJyIXr3vp9GjdpjGJ2BAcAbwBOYZiNM8wvuv38OAQHlHE4pnmSaJg888B6+vlswzYbARHKu27sd02xFREQEt9zylMMpxZsUz2XlRURERM5TmTJ+TJ78OQsXvsSCBTM5cOA9fHz8aN26F336zKVOneZORxQH1KvXkueeW828edP49ttnycxMo3z5S+jadRzXX38vgYEhTkcUL1LgIsswjH+sJ6H1JUREREqPtLSjrFu3mMzME9Sq1YTIyMudjpSvMmX86NlzDD17jsHtzsY0XXrfIoSH12P48P/SokV3UlN/Jzw8issua6/nhhS5AhdZtm0zdOhQ/PxyZmU5efIkI0eOJDAw8LR+n3zySdEmFBEREUe53dm8/fZ4Pv/8JbKyTuRur1s3hnvvfZ0aNS5zMN25uVw6cUdy3st+9tl/ee+9R0lPP5y7vWrVKEaPfpmmTa9yMJ14mwJfkzVkyBCqVKlCSEgIISEhDBo0iPDw8Nyv/7yJiIiId5kxYzj/+99/ycoaB+wFTgD/Y+fONB54oAPJybscTihybvPmTeP118eQnt4X2ApkAN9x8GAEkyZ1ZdOm5Q4nFG9i2Fqh96xSU1MJCQkhZe5cggMCnI4j4lkJCZCYyPyk5sQxiCQiCA+HqCgtRixSWuza9TP33NMUeBW47YzWw5hmI66+ugejRr3iQDqRgklLO8qQIRFkZY0Epp/RmolhtKd+fT+eemqpE/GkBElPT6V//xBSUlIIDg7Ot5/Gz0Xkn/5WXCUwkvhDTQhvXJFYFVcipU58/Du4XGG43UPyaK2AZY3g22+nMXLkizotT4qtFSs+ObUA9b/zaPXFtu9jy5b+JCfvIiystqfjiRfSq6GInC4hgfnxQSQwkkSiSCKC2D4qrkRKq6NHD2Db9YAy+fS4lMzM45w4kaY1hqTYOnr0AC5Xedzuavn0uPRUv2QVWVIkVGSJyF8SEnL+je1IYmJOVaXRK5HSrUKFCAxjETnXr/jl0eNn/PzKUbas1p2S4qtixQjc7sPAHqB6Hj1+BqBChXBPxhIvVuCJL0Sk9NG1VyJy1VU343b/DszKo/UApjmLTp1uxuVyeTqaSIG1bt0LX99AYGoerScwzWk0anQVVarU9HQ08VIqskRERCRf1as3pHv3UcA9wFhgC/A7EIdpXkG5cj7ceON4RzOKnEtAQDluueUJ4GVgEPAjcAj4HMPoiMu19VS7SNHQ6YIiIiJyViNGPE9oaBXmzXuWEyeeyd3eqNHVjBr1MpUqXeJgOpGCufbaUZQp409c3CSOHp2Tu71mzebceecS6tVr5WA68TYqskREROSsTNOkf/+J9Or1bzZuXEpm5glq1mxEeHg9p6N5pczMk2RmniAgIATT1ElHRemaa4bRqdMQ1q9fwpEj+6lRoxF167bAMAyno4mXUZElIiIiBeLnF0CLFl2djuG1EhMTeP/9x/nxx/nYtkW5clXp1u02evceR0BA/uvxSMHt3r2BDz54nB9++BjLyqJs2fJ06XIrN944nuDgSk7HEy+ij0dEREREHLZ69QLGjWvHmjXbse1ngQ85duxffPTRczzwQCzHj6c4HbHE27RpOWPHtuaHH1ZjWY8DH3LixK189tnrjB3bliNHDjgdUbyIiiwRERERB2VkpDN9+s3Ydncsay1wN3AjMAPL+oE9e7Yzd+7DDqcs2dxuN08/PYjs7FZY1s/kLEp8IzANy1rD77+n8Oab9zucUryJiiwRERERBy1f/gHp6Uex7Wf456LPTbCskXz11ZtkZp50Ip5X+OmnRRw69Cu2PR0IOKM1Esv6N8uWvU9q6iEn4okXUpElIjkSEiAxkfmJ9YmLjyApyelAIiKlw+7dG3C56gGR+fTowokTRzl8WC/M5+vXXzdgmuWBFvn0uAa3O5P9+7d7MpZ4MU18IVLa/VlcJTUngZHEH2pCeOOKxGohYhERj/DzCwCOAG4gr0WdfwfA17esB1N5Fz+/AGw7HUjnnyNZAH8Aeoyl6GgkS6Q0i4tjfnwQE5JGEscg4ulIbJ+KDBqkAktExFNat74Bt/t34NM8Wm0M41Xq1ImmQoVqHk7mPVq16oFtZwLv5NNjFlWq1KFmzUaejCVeTCNZIqVVQgJERZFALxKJAmDcIIcziYiUQnXrtqBp0y5s2DAcy/IHupHzOfgRYAK2/Q39+n3qaMaSLiysNldeOYDly+/DskLJmfTCBRwDngQ+oG/f17QumRQZFVkiQlSU0wlEREq3Bx6Yy5Qpvfnll+twuWph29Ww7XUYRjYjRrxI69Y9nY5Y4t111yxOnEhj9er+uFwR2HYNYCO2fZwBAx7lmmuGOR1RvIiKLBERERGHBQWF8vjjS9i0aTnff/8hJ04cIyLiejp1Gkr58mFOx/MKfn4BTJjwKdu2rWbp0vc4duwwYWFd6dRpKJUr13A6nngZFVkiIiIO+OKLV5g3bxpHj/6Br68vMTHXc8stTxMUFOp0NHFIZuYJkpK2sW/fdtLTj2EYLn7//TcVWUWsXr1W1KvXyukY4uVUZImI
```

Comment:
I'm not sure I get the point of this 90 degrees part

Reply:
It's a fine-tuning example. Also showing that what we really have is a training circuit that can adapt to different dataset.

Reply:
ah I see thanks ! worth emphasizing a bit on this in the text then !

Reply:
I added a cell right before the fine-tuning